presentation/frontend/package.json

```diff
@@ -54,6 +55,7 @@
     "@itwin/core-frontend": "workspace:*",
     "@itwin/core-i18n": "workspace:*",
     "@itwin/core-quantity": "workspace:*",
+    "@itwin/ecschema-metadata": "workspace:*",
```

Comment:
Do we really need these deps on `presentation-frontend`? Seems like having them on `presentation-common` should be sufficient.

Reply:
removed from backend too.